ViewModel/Inventory/OccupancyTypes/ValueUncertaintyVM.cs

```diff
@@ -87,20 +93,14 @@ public ValueUncertaintyVM(ContinuousDistribution valueUncertaintyOrdinate)
             SelectedDistributionTypeChanged(valueUncertaintyOrdinate.Type);
         }
 
-        public ValueUncertaintyVM(XElement uncertElem)
-        {
-            //read the base xml data
-            ContinuousDistribution cd = new Normal();
-        }
-
         #endregion
 
         public abstract XElement ToXML();
   
 
         public ContinuousDistribution CreateOrdinate()
         {
-            //the currentVM can equal null. That is the deterministic case
+            //the currentVM can equal null. That is the non ratio deterministic case
             if (CurrentVM == null)
             {
                 return new Deterministic(0);
```

Comment:
how is the returned deterministic distribution used?

Reply:
CreateOrdinate gets called when writing to xml. This happens when it gets saved and also to clone the object when opening for edit.
 It also get called when I translate the VM occtype objects into Model occtype objects for the stage damage compute. This is where there was a bug and I was not handling the new ratio value correctly. I'm glad you commented so that I could find that bug. Let me know if you want to know more. All of the occtype code is tough to slog through as there is so much going on.